client/public/config/configuration.json

```diff
@@ -20,6 +20,7 @@
     "DEV_EXISTING_NR_TIMEOUT_MS": 0,
     "AUTH_TOKEN_URL": "https://dev.oidc.gov.bc.ca/auth/realms/fcf0kpqr/protocol/openid-connect/token",
     "QUICK_SEARCH_PUBLIC_ID": "",
-    "QUICK_SEARCH_PUBLIC_SECRET": ""
+    "QUICK_SEARCH_PUBLIC_SECRET": "",
+    "HOTJAR_ID": "2284327"
```

Comment:
We try to not include IDs in the local config (since it's committed to GH). This should be in the config maps only.
(And maybe different IDs for dev/test/prod ?)

Reply:
This id is for local site. All other environments are using different id.

Reply:
Oh OK.

Will you show us how to use Hotjar at the next tech review?